executor/seqtest/prepared_test.go

```diff
@@ -246,7 +246,7 @@ func (s *seqTestSuite) TestPrepared(c *C) {
 		_, err = tk.Exec("execute stmt")
 		c.Assert(err, IsNil)
 		tk.MustQuery("select a from prepare1;").Check(testkit.Rows("6"))
-		_, err = tk.Exec("prepare stmt FROM @Sql")
+		_, err = tk.Exec("prepare stmt FROM @SQL")
```

Comment:
I guess `Sql` is by design here.

Reply:
Sorry I didn't get it, could you please elaborate a little bit, Thanks!

Reply:
This case wants to test an upper case char in `sql`, so it uses `Sql`.

Reply:
FYI: https://github.com/pingcap/tidb/pull/19373

Reply:
If you change it to `SQL`, then it will duplicate with L244.

Reply:
> This case wants to test an upper case char in `sql`, so it uses `Sql`.
Oh I see. Then I suggest not using `Sql` here, which is misleading, such as in this thread :D
What about changing it to this
```suggestion
		_, err = tk.Exec("prepare stmt FROM @Uppercase")
```

Reply:
ok